udata/models/__init__.py

```diff
@@ -49,6 +46,8 @@ def __init__(self, app=None):
         self.ValidationError = ValidationError
         self.ObjectId = ObjectId
         self.DBRef = DBRef
+        self.Owned = Owned
```

Comment:
This allows, as every other db mixins, to be used as `db.Owned` and `db.OwnedQuerySet`.
`db` act as a facade pattern everywhere.